src/Halogen/Query.purs

```diff
@@ -51,10 +56,21 @@ type Tell f = Unit -> f Unit
 -- | data Query a = Tick a
 -- |
 -- | sendTick :: forall o. H.HalogenIO Query o Aff -> Aff (Maybe Unit)
--- | sendTick app = app.query (H.tell Tick)
+-- | sendTick app = app.query (H.mkTell Tick)
 -- | ```
-tell :: forall f. Tell f -> f Unit
-tell act = act unit
+mkTell :: forall f. Tell f -> f Unit
+mkTell act = act unit
+
+tell
```

Comment:
I think this could use a documentation comment (maybe that's for a later PR as this becomes more stable) along the lines of the text in the original issue

Reply:
Yeah, definitely 🙂